src/chromium/cargo.md

```diff
@@ -1,56 +1,53 @@
-# Using cargo for experimental tools
+# Comparing Chromium and Cargo ecosystems
 
-Subjectively,
+Rust community typically uses `cargo` and libraries from [crates.io][2].
+Chromium is built using `gn` and `ninja` and a curated set of dependencies.
 
-```bob
-High        ^
-            |      x cargo
-            |
-Development |                     x "cargo --offline"
-speed       |
-            |                                  x  "gn/ninja"
-            |                                     "rust_executable(...)" 
-Low         +---------------------------------------------------->
-             Low               Determinism                 High
-```
-
-`cargo` works great for pure-Rust tools, but isn't optimized for large multi-
-language projects like Chromium. Chromium uses `gn` and `ninja`.
-
-When writing a tool in Rust, your choices are:
+When writing code in Rust, your choices are:
 
 * Use `gn` and `ninja` (using the `rust_executable` template we'll meet
   later)
 * Use `cargo`, but [restrict yourself to Chromium's audited toolchain and crates][0]
 * Use `cargo`, trusting a [toolchain][1] and [crates downloaded from the internet][2]
 
-Your organization's policy, and/or common sense, may prohibit you from doing
-these things.
-
 From here on we'll be focusing on `gn` and `ninja`.
 
 ## Mini exercise
 
-Discuss in small groups the policies within your own team and organization,
-and come to a group agreement about what's an acceptable level of risk.
+Discuss in small groups the policies within your own team and
+organization, and come to a group agreement about your assessment
+of risk associated with the choices above.
 
 <details>
 
-Talk about the cultural differences between the `cargo` world and the Chromium
```

Comment:
Could you put these two paragraphs back?

I think it's really important to discuss the cultural differences / the "feel" of developing in cargo vs Chromium. I think this is arguably one of the most important parts of this whole Chromium course - setting expectations for how Chromium Rust development will feel, relative to the agility obtained in a pure cargo world. I think we do a pretty great job of making it relatively easy to bring in crates, but it's still a long way from the productivity you can achieve in cargo. Our processes are optimized to make something we can safely ship to billions of people...

The second paragraph is here because I've heard from friends at Amazon, Facebook etc. that their user-base is initially surprised by the idea of writing _tools_ in Rust, but it gradually becomes clear that Rust is just as effective as Python in writing tools — and then those tools turn out to "just work" for a wider variety of inputs and corner cases. I think that's worth talking about. I don't mind if you reword it!